### DISCUSSION TITLE
how do you make extension colors sprites

### DISCUSSION THREAD

**Discussion by unknown07724** [TurboWarp/extensions#1589] [General]
To be honest it is madness

**Comment by David-Orangemoon** [comment]
Could you elaborate?

**Reply by unknown07724** [reply]
Yes I can some people make the block color a image somehow

**Comment by snowboyz0825** [comment]
Do you mean 
1. Some people can make the background of extension blocks an image and you'd like to know how
2. You'd like to know how to change the background color of custom extension blocks
3. You'd like to know how to add an "Icon" of sorts, such as what sensing+ has
also wow this comment really makes me look like a bot

**Reply by unknown07724** [reply]
1.

**Comment by Ender-Studio** [comment]
sprite tint?

**Reply by unknown07724** [reply]
no
